helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -117,6 +117,10 @@ processors:
           - set(resource.attributes["com.splunk.sourcetype"], Concat(["kube:object:", attributes["k8s.resource.name"]], ""))
   {{- end }}
 
+  {{- if and $clusterReceiver.eventsEnabled (eq (include "splunk-otel-collector.logsEnabled" .) "true") }}
```

Comment:
We actually need to update this if statement, there are cases where k8sattributes/clusterReceiver processor is not included but it should be because it is [used in pipelines](https://github.com/signalfx/splunk-otel-collector-chart/pull/1481/files/7345d03bcc7b760cfb512a068061e26f1111475f#diff-4e951abec9da3c5944c771fee7bd203aab089a25f1ea23d2b86fadc0590d7074R133).

Reply:
There might be cleaner ways of doing this, but I think we will at least something here that looks like this. See what you can do.
```
{{- if or 
  (and $clusterReceiver.eventsEnabled (eq (include "splunk-otel-collector.logsEnabled" .) "true")) 
  (and (eq (include "splunk-otel-collector.objectsEnabled" .) "true") (eq (include "splunk-otel-collector.logsEnabled" .) "true")) 
  (eq (include "splunk-otel-collector.o11yInfraMonEventsEnabled" .) "true") 
}}
```

Reply:
or we can introduce sth like this:
`{{- $splunkLogsEnabled := eq (include "splunk-otel-collector.logsEnabled" .) "true" -}}`
`{{- $splunkObjectsEnabled := eq (include "splunk-otel-collector.objectsEnabled" .) "true" -}}`
`{{- $o11yInfraMonEventsEnabled := eq (include "splunk-otel-collector.o11yInfraMonEventsEnabled" .) "true" -}}`

`{{- if or 
  (and $clusterReceiver.eventsEnabled $splunkLogsEnabled) 
  (and $splunkObjectsEnabled $splunkLogsEnabled) 
  $o11yInfraMonEventsEnabled 
}}`

but here we will need to refactor the whole file and this will introduce even more variables

Reply:
I think the current state of this PR LGTM. Let's try to get this merged in and you can do more refactoring in separate PRs as you see fit. I'll merge this work shortly.